wakebox/main.cpp

```diff
@@ -271,8 +271,9 @@ int main(int argc, char *argv[])
 			bind_ops.push_back({"bind", source, destination});
 		}
 		if (arg(options, "bind-cwd")->count > 0) {
-			bind_ops.push_back({"create-dir", "", "/tmp/workspace"});
-			bind_ops.push_back({"bind", get_cwd(), "/tmp/workspace"});
+			std::string cwd = get_cwd();
+			bind_ops.push_back({"create-dir", "", cwd});
```

Comment:
This seems an odd choice. Why would you want to keep the same path as outside wakebox?
I'd have thought the prior behavior was better default.

Reply:
I think it make it less jarring for people if the working directory is the same when they drop in to wakebox.
It won't affect any reproduction of jobs from a json file or anything like that